docs/testplan/build_2tier_network.md

```diff
@@ -0,0 +1,386 @@
+# Build A 2-Tier Network
```

Comment:
this should work for any number of tiers, so better rename the file to something like "Multi-tier DUT testbed support"